ctapipe/reco/hillas_intersection.py

```diff
@@ -218,6 +218,15 @@ def _predict(self, hillas_dict, array_pointing, telescopes_pointings=None):
         src_fov_lon, src_fov_lat, err_fov_lon, err_fov_lat = self.reconstruct_nominal(
             hillas_dict_mod
         )
+
+        # Catch events reconstructed at great angular distance from camera center
+        # and retrun INVALID container to prevent SkyCoord error below.
+        if (
+            np.abs(src_fov_lat) * u.rad > 45 * u.deg
```

Comment:
I'd define the 45 ° limit in a named constant on top, something like `FOV_ANGULAR_DISTANCE_LIMIT = 45 * u.deg` and then compare to `FOV_ANGULAR_DISTANCE_LIMIT.to_value(u.rad)` instead of multiplying both sides with the unit (faster).

Maybe you can also put this part into a function so it looks like this:

```
if far_outside_fov(src_fov_lat, src_fov_lon):
    return INVALID
```